spec/examples.txt

```diff
@@ -0,0 +1,22 @@
+example_id                                  | status | run_time        |
```

Comment:
What's this file? Is it meant to be checked in?

Reply:
no it is not, I added .examples.txt to the .gitignore but for some reason it's stuck around :(

Reply:
you will need to edit the commit and remove it before adding it to the gitignore file (I think)

Reply:
I think you'll still need to `git rm` it, `.gitignore` prevents new changes from showing up, but doesn't remove anything that's already commited.